types/result.go

```diff
@@ -182,11 +182,12 @@ func NewResponseFormatBroadcastTx(res *ctypes.ResultBroadcastTx) TxResponse {
 	parsedLogs, _ := ParseABCILogs(res.Log)
 
 	return TxResponse{
-		Code:   res.Code,
-		Data:   res.Data.String(),
-		RawLog: res.Log,
-		Logs:   parsedLogs,
-		TxHash: res.Hash.String(),
+		Code:      res.Code,
+		Codespace: res.Codespace,
+		Data:      res.Data.String(),
+		RawLog:    res.Log,
+		Logs:      parsedLogs,
+		TxHash:    res.Hash.String(),
 	}
 }
 
```

Comment:
Should we consider writing the codespace in `String()`?

Reply:
Both of `TxResponse.Codespace` and `ResultBroadcastTx.Codespace` are string type... (?)

Reply:
No, they're integers.

```go
	if r. Codespace > 0 {
		sb.WriteString(fmt.Sprintf("  Codespace: %d\n", r. Codespace))
	}
```

Reply:
I can't find that code. 

https://github.com/cosmos/cosmos-sdk/blob/master/types/result.go#L224

Reply:
Right, I'm asking you to add it ;-)

Reply:
yes, it is already added.